instaclustr/resource_vpc_peering.go

```diff
@@ -187,3 +199,25 @@ func resourceVpcPeeringStateImport(d *schema.ResourceData, meta interface{}) ([]
 	d.Set("vpc_peering_id", idParts[1])
 	return []*schema.ResourceData{d}, nil
 }
+
+func createVpcPeeringRequest(d *schema.ResourceData) (CreateVPCPeeringRequest, error) {
+	result := CreateVPCPeeringRequest{}
+	if _, isSet := d.GetOk("peer_subnet"); isSet {
+		result = CreateVPCPeeringRequest{
```

Comment:
Could we reduce repetition here? I.E define the base CreateVPCPeeringRequest and set PeerSubnet/PeerSubnets appropriately?